api/_lib/examples/example-list.ts

```diff
@@ -1,21 +1,25 @@
-// Currently we read & parse the README file from zeit/now-examples
-// TODO: create a `manifest.json` for zeit/now-examples
+import Frameworks, { Framework } from '../../../packages/frameworks';
 
-import fetch from 'node-fetch';
-
-/**
- * Fetch and parse the `Frameworks and Libraries` table
- * in the README file of zeit/now-examples
- */
-export async function getExampleList() {
-  const response = await fetch(
-    `https://raw.githubusercontent.com/zeit/now-examples/master/manifest.json`
-  );
-
-  if (response.status !== 200) {
-    console.log('manifest.json missing in zeit/now-examples');
-    return null;
-  }
+interface Example {
+  example: string;
+  path: string;
+  demo: string;
+  description: string;
+  tagline: string;
+  framework: string;
+}
 
-  return response.json();
+export async function getExampleList(): Promise<Example[]> {
+  return (Frameworks as Framework[])
```

Comment:
What's the purpose of `as` here? Is this still an issue with `@now/frameworks` package?

Reply:
@styfle yes, it's still a JSON file, I've tried to convert it, but ran into some problems, so I couldn't convert it yet